composer.json

```diff
@@ -14,13 +14,18 @@
         "symfony/framework-bundle": "~2.7|~3.1",
         "symfony/finder": "~2.7|~3.1",
         "symfony/console": "~2.7|~3.1",
-        "symfony/intl": "~2.7|~3.1"
+        "symfony/intl": "~2.7|~3.1",
+        "symfony/asset": "~2.7|~3.1",
+        "symfony/templating": "~2.7|~3.1",
+        "symfony/translation":  "~2.7|~3.1",
+        "doctrine/annotations": "~1.0"
```

Comment:
This bundle does not rely on annotations, so there is no reason to make them mandatory requirements